pkg/azure/README.md

```diff
@@ -84,24 +95,24 @@ Follow these steps to ensure consistent structure and seamless integration.
          logger.Log.Printf("An error occurred while getting resource ID info: %+v", err)
          return nil, err
      }
-     err = s.setupAzureHandler(resourceIdInfo)
+     // the namespace parameter could be de different from req.Deployment.Namespace
+     handler, err = s.setupAzureHandler(resourceIdInfo, req.Deployment.Namespace)
      if err != nil {
          return nil, err
      }
+     // Use handler
      ```
 
 3. Handling ARM Requests:
-   - If your new API needs to make ARM requests, check if the required functionality is already implemented in `sdk_handler.go`. If not, add a new function in the handler that wraps the Azure request. Call this new function from the `plugin` file. Any helper functionalities relevant only to Paraglider and the server can remain in `plugin`.
+   - If your new API needs to make ARM requests, check if the required functionality is already implemented in `sdk_handler.go`. If not, add a new function in the handler that wraps the Azure request. Call this new function from the `plugin` file. Ideally, helper functionalities relevant to Paraglider and the server should be outside `plugin`.
```

Comment:
Could we say "Ideally, only the Paraglider plugin interface RPCs should be in `plugin` and helper functions should go in the appropriate file" or something like that?

Reply:
Updated